buyer-api/test/facades/dataResponseFacade/addDataResponse.test.js

```diff
@@ -1,13 +1,121 @@
 import test from 'ava';
-// import { addDataResponse } from '../../../src/facades';
-
-test.skip('responds with error when orderAddress is invalid', t => t.fail());
-test.skip('responds with error when sellerAddress is invalid', t => t.fail());
-test.skip('responds true if seller has already accepted', t => t.fail());
-test.skip('throws if it cannot retrieve data response from storage', t => t.fail());
-test.skip('throws if notaryAccount is not an address', t => t.fail());
-test.skip('throws if dataHash is undefined', t => t.fail());
-test.skip('throws if signature is undefined', t => t.fail());
-test.skip('throws if notary is not added to dataorder', t => t.fail());
-test.skip('throws if buyer does not have enough allowance to add DataResponse', t => t.fail());
-test.skip('returns true if dataResponse is successfully added', t => t.fail());
+import sinon from 'sinon';
+import * as s3 from '../../../src/utils/wibson-lib/s3'; // { getDataResponse }
+import * as utils from '../../../src/utils'; // { dataOrderAt }
+
+// addDataResponse(order, seller, enqueueTransaction)
+import { addDataResponse } from '../../../src/facades';
+
+const fakeAddress = '0x1234567890123456789123456789012345678901';
+const fakeSignature = '0x1234567890123456789123456789012345678901123456789012345678912345678901234567890112345678901234567891234567890123456789011234567890';
+
+test.afterEach.always(() => { sinon.restore(); });
+
+test.serial('responds with error when orderAddress is invalid', async (assert) => {
```

Comment:
I would change the title to `throws an error when orderAddress is invalid` so it there are no possible misunderstandings.